net.openhabittracker.OpenHabitTracker.yaml

```diff
@@ -0,0 +1,53 @@
+id: net.openhabittracker.OpenHabitTracker
+#runtime: org.freedesktop.Platform
+#runtime-version: '24.08'
+#sdk: org.freedesktop.Sdk
```

Comment:
```suggestion
```

let's drop these if you need webkit2gtk.

Reply:
ok, done